crates/trigger/src/runtime_config/sqlite.rs

```diff
@@ -53,6 +52,7 @@ fn execute_statements(
 #[serde(rename_all = "snake_case", tag = "type")]
 pub enum SqliteDatabaseOpts {
     Spin(SpinSqliteDatabaseOpts),
+    Turso(TursoOpts),
```

Comment:
Thoughts vs. naming this a `libsql` backend?

Reply:
I don't think it's really about libsql as that can be run locally and should work fine with the default spin configuration. Perhaps `sqld` or `remote_libsql`?

Reply:
The library is `libsql-client`. LibSQL is intended to support both local scenarios and remote scenarios, depending on the URI, so I feel like calling it `libsql` wouldn't be inappropriate.

Reply:
Should we also allow running libsql locally as well?